<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenu</class>
 <widget class="QWidget" name="GameMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit
{
	 font-family: &quot;Microsoft YaHei&quot;;
	 font-size:10pt;
	 font-weight: bold;
}</string>
  </property>
  <widget class="QLineEdit" name="serverPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>Port端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serverStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 40px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:11pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 40px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
 }</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientPort">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Port端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientStart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 60px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 60px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:9pt;
	  font-weight: bold;
	  color:white;
 }</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>IP地址</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
